aictl/cli/main.py

```diff
@@ -319,6 +358,18 @@ def main():
     t2i_parser.add_argument( "-o", "--output-path", default=f"output_t2i{utc_time}.png", help="path for image output when generation is complete")
     t2i_parser.set_defaults(func=t2i)
 
+    classify_parser = subparsers.add_parser("classify", help="Image classifier")
```

Comment:
can you make this description more consistent with the other help command text, like:
"an image classification subcommand" and "an image segmentation subcommand"